packages/datatrak-web/src/api/queries/useProjectUsers.ts

```diff
@@ -0,0 +1,18 @@
+import { useQuery } from '@tanstack/react-query';
+import { Project, DatatrakWebUsersRequest } from '@tupaia/types';
+import { get } from '../api';
+
+export const useProjectUsers = (projectCode?: Project['code'], searchTerm?: string) => {
```

Comment:
Thoughts on picking up the naming convention where these all end in `...Query`?

```suggestion
export const useProjectUsersQuery = (projectCode?: Project['code'], searchTerm?: string) => {
```